build/bp-activity/bp-activity-blocks.php

```diff
@@ -380,7 +380,7 @@ function bp_activity_render_share_activity_block( $attributes = array() ) {
 		array(
 			'text'                => '',
 			'borderRadius'        => '',
-			'style'               => '',
+			'style'               => array(),
```

Comment:
I changed this into an `array` since this is what's expected and it was causing `undefined` errors.